<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChooseMenu</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Desktop/newspaper.png</normaloff>../../../../Desktop/newspaper.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="num_pushButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>780</y>
     <width>260</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>돌아가기 : 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_view">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>700</width>
     <height>650</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>700</width>
     <height>650</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;나눔스퀘어&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'나눔스퀘어'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>780</y>
     <width>260</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>현재기사 : 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_pushButton_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>780</y>
     <width>260</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;나눔스퀘어&quot;;
background-color: rgb(227, 227, 255);</string>
   </property>
   <property name="text">
    <string>다음 키워드 : 3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
